src/main/java/core/basesyntax/service/RegistrationService.java

```diff
@@ -3,5 +3,8 @@
 import core.basesyntax.model.User;
 
 public interface RegistrationService {
+
     User register(User user);
+
+    boolean clearStorage();
```

Comment:
Make sure to not add something that is not described in the task requirements

see common mistakes (https://mate-academy.github.io/jv-program-common-mistakes/java-core/junit/registration-validation)

Reply:
I was instructed by Anastasia to clean storage each time, and to have only one object of service, which is why it is announced one time before all tests